### PR TITLE
docs: add frontend meeting notes 2021-02-17

### PR DESCRIPTION
**Pull request by amcgee**
I'm not sure the attendees list is correct, please let me know if I missed you

### CODE REVIEW

**Review by mediremi — APPROVED** [review]

**Review by paschalidi — APPROVED** [review]
Thanks for keeping the notes Austin

File: decisions/2021/02/17-fe-meeting.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+
+* Cypress
+    * Want to deprecate - getWithDataTest / findWithDataTest
+    * Use helpers instead of commands
+    * Need to document
+* PWA/Offline/Responsive
+    * PWA deferred to after 2.36
+        * Beta on App Hub in Q1 2021
+* Responsive Working on control bar + portrait / tablet
```
Comment (varl) [inline decisions/2021/02/17-fe-meeting.md:32]: This bullet seems to belong under "PWA/Offline/Responsive":

```suggestion
    * Responsive Working on control bar + portrait / tablet
```

**Review by varl — APPROVED** [review]
:tada:

**Comment by varl** [comment]
> Thanks for keeping the notes Austin

Yes, thank you!

**Review by Mohammer5 — APPROVED** [review]

**Review by cooper-joe — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**